include/types.h

```diff
@@ -58,6 +60,8 @@ typedef unsigned long	uintptr_t;
 typedef unsigned long	size_t;
 typedef long		ssize_t;
 
+typedef int		bool;
```

Comment:
:'(  This is buggy in various cases, because int and _Bool have different properties for nonzero values.

Include stdbool, or typedef against _Bool, but never try and make `bool` a regular char/int because it does go wrong in subtle ways

Reply:
Fixed